src/parquet/arrow/schema.h

```diff
@@ -85,6 +85,8 @@ ::arrow::Status PARQUET_EXPORT ToParquetSchema(const ::arrow::Schema* arrow_sche
                                                const WriterProperties& properties,
                                                std::shared_ptr<SchemaDescriptor>* out);
 
+int32_t DecimalSize(int32_t precision);
```

Comment:
I presume this does not need to be exported. It could go in a private header, also

Reply:
No, this shouldn't be available to third parties.